scripts/rebase.sh

```
@@ -0,0 +1,551 @@
+
+    is_clean=$(git status | tail -n 1)
+    if [ "$is_clean" != "nothing to commit, working tree clean" ]; then
+        echo -e "${RED}Cannot rebase! There are uncommited changes:"
+        git_status
+        exit 1
+    fi
+
+
+    ### Select branch which will become a base
+    if [ -n "$main" ]; then
+        if [ "$current_branch" == "${main_branch}" ]; then
```

Shellcheck (0.10.0)

[warning] current_branch is referenced but not assigned.

(SC2154)